MonoGame.Framework/Graphics/GraphicsDevice.OpenGL.cs

```diff
@@ -88,11 +88,15 @@ internal void SetVertexAttributeArray(bool[] attrs)
             }
         }
 
-        private void PlatformSetup()
+       private void PlatformSetup()
         {
 #if WINDOWS || LINUX || ANGLE
             GraphicsMode mode = GraphicsMode.Default;
-            var wnd = (Game.Instance.Window as OpenTKGameWindow).Window.WindowInfo;
+
+            var wnd = OpenTK.Platform.Utilities.CreateWindowsWindowInfo(PresentationParameters.DeviceWindowHandle);
```

Comment:
@cra0zy - The point would be to not use `Game.Instance`.  We should be able to get what we need from `PresentationParameters.DeviceWindowHandle`.  `GraphicsDevice` needs to be independent from `Game`.